pallets/pallet-asset-switch/README.md

```diff
@@ -1 +1,129 @@
+
+## In a gist
+
+The pallet lets users on the source chain lock `N` tokens of the chain's native currency into a switch pair specific account, to receive `N` remote assets on the configured remote location, which are transferred to them from the source chain's sovereign account at destination.
+Then, the chain, which i
+
+## Design choices
+
+The pallet aims to be generic enough for most parachains to be able to add one or more instances of the asset switch pallet into their runtime, after configuring it according to their needs.
+
+The pallet makes the following assumptions:
+
+* The local asset to switch is always the chain local currency, implementing the `fungible::Mutate` trait.
+* The switch ratio is pre-defined to be 1:1, i.e., one unit of local currency for one unit of remote asset.
+* The pallet only exposes calls to allow local -> remote switches. The reverse is assumed to happen via XCM reserve transfer from the configured remote location, for which the pallet crate provides all the XCM components that are dynamically configured based on the switch pair information stored within each instance of this pallet.
+* The sovereign account of the source chain at destination owns the remote assets in the amount that is specified when the switch pair is created. The validation of this requirement is delegated to the source chain governance in the act leading to the creation of the switch pair, as the source chain itself has currently no means of doing that.
+* Similarly, the pallet has currently no way to verify that a transfer of remote tokens from the chain sovereign account to the specified beneficiary has been completed successfully on the remote location, hence it takes an optimistic approach given that all the verifiable preconditions for the switch are verified on the source chain, i.e., the chain from which the switch originates. Any unexpected issues in the transfer on the remote location will most likely require intervention of the source chain governance to re-balance the state and make it consistent again.
+
+## Add the pallet to the runtime
+
+Add the following line to the runtime `Cargo.toml` dependencies section:
+
+```toml
+pallet-asset-switch = {git = "https://github.com/KILTprotocol/kilt-node.git", branch = "release-1.13.0"}
+```
+
+The asset switch pallet is available in the KILT node release 1.13.0 and later.
+
+## Configure the pallet
+
+The pallet can be added one or more times to the runtime.
+
+For multiple deployments of the same pallet (e.g., to bridge the local currency to different remote assets), pass runtime configuration to the pallet's `Config` trait.
+
+```rust
```

Comment:
```suggestion
```rust,ignore
```
in rust 1.74.0 he is trying to compile the examples, which will fail.

Reply:
Even in the README? Even for a simple `Cargo.toml` example?

Reply:
If the README is included in the source code, yes. If I remember correctly, rust is not checking TOML files.

Reply:
https://github.com/KILTprotocol/kilt-node/pull/667/commits/6e080b127b8f97b9b291088008afa7c38cf3b7d2